addon/i3dio/node_classes/skinned_mesh.py

```diff
@@ -78,7 +85,28 @@ def add_i3d_mapping_to_xml(self):
 
     def _add_bone(self, bone_object: bpy.types.Bone, parent: Union[SkinnedMeshBoneNode, SkinnedMeshRootNode]):
         """Recursive function for adding a bone along with all of its children"""
-        self.bones.append(self.i3d.add_bone(bone_object, parent))
+        is_child_of = False
+
+        # Try to find pose bone for the bone object
+        if pose_bone := self.blender_object.pose.bones.get(bone_object.name):
+            processed_objects = self.i3d.processed_objects
+
+            # Check if pose pose bone has a CHILD_OF constraint
+            child_of = next((c for c in pose_bone.constraints if c.type == 'CHILD_OF'), None)
+            if child_of and child_of.target:
+                target = child_of.target
+                if target not in processed_objects:
+                    # Target object has not been processed yet, defer the constraint
+                    self.logger.debug(f"Deferring CHILD_OF constraint for {bone_object.name} -> {target}")
+                    self.i3d.deferred_constraints.append((bone_object, target, None))

```

Comment:
```suggestion
                    self.i3d.deferred_constraints.append((bone_object, target))
```
This extra `None` makes everything fail, to me it looks like a merge error? Because I cannot see why it should be there.